charts/lagoon-logs-concentrator/ci/linter-values.yaml

```diff
@@ -2,6 +2,7 @@
 elasticsearchHost: "logs-db-service.elasticsearch.svc.cluster.local"
 elasticsearchAdminUser: "admin"
 elasticsearchAdminPassword: "securepass"
+defaultIngressClass: nginx
```

Comment:
minor nit: should we make this `defaultIngressClassName` for consistency with the other option under each ingress?

Reply:
fixed